mantle/rbx_mantle/src/state/aws_credentials_provider.rs

```diff
@@ -16,6 +19,8 @@ impl AwsCredentialsProvider {
             prefixed_environment_provider: EnvironmentProvider::with_prefix("MANTLE_AWS"),
             environment_provider: EnvironmentProvider::default(),
             profile_provider: ProfileProvider::new().ok(),
+            container_provider: ContainerProvider::default(),
```

Comment:
I haven't tested this out locally but based on past experience and the rusoto docs, it looks like both of these providers have a 30 second timeout by default. This is not a great experience for most users since causes Mantle to hang for 30 seconds before determining there are missing credentials. Maybe this could be an opt-in feature via an environment variable?